matrix/all-in-one.yaml

```diff
@@ -0,0 +1,575 @@
+    default: '1200'
+    type: int
+    category: instrumentation
+    to_do_comment: int?
+  - name: OTEL_DOTNET_AUTO_GRAPHQL_SET_DOCUMENT
+    description: Whether the GraphQL instrumentation can pass raw queries as a `graphql.document` attribute. As queries might contain sensitive information, the default value is `false`.
+    default: 'false'
+    type: boolean
+    category: instrumentation
+  - name: OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_LEGACY_SOURCES
+    description: Comma-separated list of additional legacy source names to be added to the tracer at the startup. Use it to capture `System.Diagnostics.Activity` objects created without using the `System.Diagnostics.ActivitySource` API.
+    default: ''
+    type: string
+    category: instrumentation
+  - name: OTEL_LOG_LEVEL
+    description: Sets the logging level for instrumentation log messages. Possible values are `none`, `error`, `warn`, `info`, and `debug`. The default value is `info`. Can't be set using the web.config or app.config files.
+    default: 'info'
+    type: enum
+    category: diagnostic logging
+    to_do_comment: handle enum values/mark as string
+  - name: OTEL_DOTNET_AUTO_LOG_DIRECTORY
+    description: Directory of the .NET tracer logs. The default value is `/var/log/opentelemetry/dotnet` for Linux, and `%ProgramData%\OpenTelemetry .NET AutoInstrumentation\logs` for Windows. Can't be set using the web.config or app.config files.
+    default: 'see descruotuin'
+    type: string
+    category: diagnostic logging
+    to_do_comment: defaultvalue
+  - name: OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED
+    description: Whether the traces console exporter is activated. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED
+    description: Whether the metrics console exporter is activated. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED
+    description: Whether the logs console exporter is activated. The default value is `false`.The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_LOGS_INCLUDE_FORMATTED_MESSAGE
+    description: Whether the log state have to be formatted. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - name: MANUAL SETTINGS
+    description: How to handle https://docs.splunk.com/observability/en/gdi/get-data-in/application/otel-dotnet/configuration/advanced-dotnet-configuration.html#environment-variables-for-manual-installation?
+    default: 'N/A'
+    type: N/A
+    category: manual
+    to_do_comment: See description
+instrumentations:
+  - keys:
+      - ASPNET
```

Comment:
for Go there is no `key`.

Reply:
@Kielek What's the purpose of this field again?

Reply:
https://docs.splunk.com/observability/en/gdi/get-data-in/application/otel-dotnet/dotnet-requirements.html - Instrumentation ID. It can be used in evn. vars. to enable/disable in granulary.

Similar thing for resource detectors